bigquery/api/SimpleApp/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ * Copyright (c) 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START bigquery_simple_app_all]
+// [START bigquery_simple_app_deps]
+
+using System;
+using Google.Cloud.BigQuery.V2;
+// [END bigquery_simple_app_deps]
+
+namespace GoogleCloudSamples
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            string projectId = "YOUR-PROJECT-ID";
+            // [START bigquery_simple_app_client]

```

Comment:
Shouldn't the `projectId` line be included here so that folks can see that it is a string when they create the client?

Reply:
+1